projects/smoke_adt7476a/src/main.c

```diff
@@ -0,0 +1,76 @@
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
+#include "gpio.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define SET_SPEED_INTERVAL 5
+#define FAN_SPEED_INCREMENT 10
+#define I2C_WRITE_ADDR_1 0x5E
+#define I2C_WRITE_ADDR_2 0x58
+#define I2C_READ_ADDR_1 0x5F
+#define I2C_READ_ADDR_2 0x59
+
+// 2 storages for 2 components
```

Comment:
I think we should actually only test one fan controller at a time, because there are some boards (or at least there will be some boards) such as power distribution that will only use one fan controller